upgrade-advisor.php

```diff
@@ -25,8 +25,15 @@ function isThere($setting) {
         }
     }
 
+    $googleapi_setttings = json_decode(get("https://maps.googleapis.com/maps/api/geocode/json?key=" .$google_maps_api_key. "&address=91409"));
+
     $root_server_settings = json_decode(get(getHelplineBMLTRootServer() . "/client_interface/json/?switcher=GetServerInfo"));
```

Comment:
Can you move this line below the block below?  Since it does not make sense to make an HTTP request that you may not even need to make, you might as well not use it until you need it.